main.go

```diff
@@ -205,14 +205,13 @@ func main() {
 
 func checkOsSignals(mountpoint string) {
 	signals := make(chan os.Signal, 1)
-	signal.Notify(signals, syscall.SIGINT)
+	signal.Notify(signals, syscall.SIGINT, syscall.SIGTERM)
 
 	go func() {
 		for sig := range signals {
-			if sig == syscall.SIGINT {
```

Comment:
This check wasn't needed, since the channel only every receives the signals it specified via `signal.Notify()`.